docs/presentation/Hierarchies/ChildNodeRule.md

```diff
@@ -8,18 +8,18 @@ The rules have two types of attributes: for defining *placement* and for definin
 
 ## Attributes
 
-| Name                                                        | Required? | Type                                              | Default |
-| ----------------------------------------------------------- | --------- | ------------------------------------------------- | ------- |
+| Name                                                        | Required? | Type                                                                 | Default |
```

Comment:
This is a note for line 7 (I could not place comment on that line, Github only seems to let you comment on lines with changes).

Visually italic font does a poor job of highlighting.
![image](https://user-images.githubusercontent.com/89905873/148237904-6ad214aa-49ba-4358-8f43-65fbc8e95e8c.png)

Maybe we could use something like **bold text**?

Reply:
After looking at the doc site live, I think we should leave them as is. The text in italic in that sentence is to distinguish a term from the rest of the text, but I believe the emphasis is not strong enough to require bold text.